src/EnergyPlus/PlantUtilities.hh

```diff
@@ -158,10 +141,8 @@ namespace PlantUtilities {
                                                      bool FirstHVACIteration);
 
     void InterConnectTwoPlantLoopSides(EnergyPlusData &state,
-                                       int Loop1Num,
-                                       DataPlant::LoopSideLocation Loop1LoopSideNum,
-                                       int Loop2Num,
-                                       DataPlant::LoopSideLocation Loop2LoopSideNum,
+                                       PlantLocation const &Loop1PlantLoc,
+                                       PlantLocation const &Loop2PlantLoc,
```

Comment:
Functions in which the individual Plant Location variables were not a return value were converted to use the Plant Location variable as a const reference.